yellowfinLib.py

```diff
@@ -683,6 +698,31 @@ def butter_lowpass_filter(data, cutoff, fs, order):
     # y = filtfilt(b, a, data)
     return output
 
+def skiprows(fldrlistPPK):
```

Comment:
please rename to pep8 convetion (uses _)  for example `skip_rows` .  I'd also suggest being more specific what  tile this is assuming structure of... for example a name like `identify_first_datarow_posfile` might be an explicit function name